docs/en/engines/database-engines/mysql.md

```diff
@@ -51,6 +51,20 @@ All other MySQL data types are converted into [String](../../sql-reference/data-
 
 [Nullable](../../sql-reference/data-types/nullable.md) is supported.
 
+## Global Variables Support {#global-variables-support}
+
+For better compatibility you may address global variables in MySQL style, as `@@identifier`. 
+
+By now only these variables are supported:
+- `version`
+- `max_allowed_packet`
```

Comment:
Need to mention that they are stubs and don't correspond to anything.

Reply:
Done